src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      class="box__search1"
+      data-qa="big"
+    >
```

Comment:
The class name `box__search1` does not represent the meaning of the content and seems to indicate a style or tag name. Consider using a class name that reflects the function or content of the form.